src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -234,7 +248,7 @@ def add(
 
         # Add instance views to memory
         instance = instance_image.clone()
-        instance[~valid_depth] = -1
+        # instance[~valid_depth] = -1
```

Comment:
dead code/how should we handle this? why is this removed?

Reply:
If we apply valid_depth at this stage, then instance_image is 1d (n_kept) instead of 2d (h, w)
We probably want to keep the instance mask and the valid mask around, since we want the instance mask for masking out the RGB and both of them for masking out the pointcloud

Reply:
This was removed because we handle the masking later, in process_instances_for_env

Reply:
I think it's still useful to handle it earlier here, for navigation?

Reply:
I guess this actually is irrelevant to that though so sgtm